doc/man7/property.pod

```diff
@@ -126,16 +126,13 @@ Two levels of property query are supported.
 A context based property query that applies to all fetch operations and a local
 property query.
 Where both the context and local queries include a clause with the same name,
-the local clause is used and the context one ignored.
-For example, a context property query of "fips=yes" and a local property query
-of "fips=no" would result in algorithms that have the "fips" property set t
-"no".
+the local clause overrides the context clause.
 
 =head2 Override
 
-It is possible for a local property query to override a clause in the context
+It is possible for a local property query to remove a clause in the context
 property query by preceeding the property name with a '-'.
```

Comment:
(An alternative would be to use the tilde instead of the minus sign for removal:  `~prop`  instead of `-prop`)

Reply:
`~` would be a good `=false` equivalent at some stage in the future.  (alternatively `!`)

`\` is one of the common symbols used for set removal/subtraction in mathematics.  (`-` is the other one which is the symbol being replaced).  Perhaps even `/` to save backslash for future use.

Thoughts anyone?

Reply:
Well, I can live with the minus sign. The tilde was just a thought.

But you are right `?~prop` looks much better than `?!prop`, so let's better reserve the tilde for the unary negation operator. :-)

Reply:
?! has another [meaning](https://en.wikipedia.org/wiki/Chess_annotation_symbols#?!_(Dubious_move)).

Reply:
The more syntax you have now, "for convenience" the more you limit the future, having to work around things.  Almost all uses will be "fips=yes" or "dont care about fips"  Support that, only that, in this release.

Reply:
It is fairly minimal at present.  The only extra being the "optional" operator.